packages/react-components/react-field/src/components/Field/useField.tsx

```diff
@@ -3,7 +3,7 @@ import { CheckmarkCircle12Filled, ErrorCircle12Filled, Warning12Filled } from '@
 import { Label } from '@fluentui/react-label';
 import { getNativeElementProps, resolveShorthand, useId } from '@fluentui/react-utilities';
 import type {
-  FieldComponent,
+  FieldControl,
   FieldConfig,
```

Comment:
_nit:_ Keep these ordered alphabetically

```suggestion
  FieldConfig,
  FieldControl,
```